wasm/main.py

```diff
@@ -1871,14 +1875,14 @@ def spec_t2cvtopt1(config):
 
     S = config["S"]
     instr = config["instrstar"][config["idx"]][0]
-    t2 = instr[0:3]
-    t1 = instr[-3:]
+    t2 = ValType.from_str(instr[0:3])
+    t1 = ValType.from_str(instr[-3:])
```

Comment:
I guess `[-3:]` is open ended in the same way that `[:3]` is open ended but it's guaranteed to return no more than a length-3 string the same as `[0:3]` or `[:3]`.

Also, this `instr` value is something like `f32.convert/i32` which is something that will get converted to a formal data structure of some kind at some point soon so this can be viewed as temporary code.

Reply:
Yeah, that was dumb, I missed the `-`. :+1: for coming back to it later, if necessary.